obspy/core/inventory/network.py

```diff
@@ -407,13 +439,23 @@ def select(self, station=None, location=None, channel=None, time=None,
                 if not sta.is_active(time=time, starttime=starttime,
                                      endtime=endtime):
                     continue
+            geo_filters = (minlatitude, maxlatitude,
+                           minlongitude, maxlongitude,
+                           latitude, longitude,
+                           minradius, maxradius)
+            if not inside_geobounds(sta, *geo_filters):
```

Comment:
so this is called even if all items in ``geo_filters`` are ``None`` ? It could maybe be skipped?